git-filter-repo

```diff
@@ -2915,10 +2915,11 @@ class RepoAnalyze(object):
     stats = RepoAnalyze.gather_data(args)
 
     # Write the reports
-    sys.stdout.write(_("Writing reports to %s...") % decode(reportdir))
+    sys.stdout.write(_("Writing reports to \"%s\"...") % decode(reportdir))
     sys.stdout.flush()
     RepoAnalyze.write_report(reportdir, stats)
-    sys.stdout.write(_("done.\n"))
+    sys.stdout.write(_(" done.\n"))
+    sys.stdout.write(_("README: \"%s\"\n") % decode( os.path.join(reportdir, b"README") ))
```

Comment:
Not sure whether this belongs more as a parenthetical on the line where we mention writing reports, or if this is fine.  I guess it's okay as-is.

Reply:
As you wish, I used two separate lines to prevent paths getting cut due to terminal width.
Seems more compatible to avoid breaking ctrl+clicking, but in fact vscode can handle it (don't know other editors etc):
![image](https://github.com/user-attachments/assets/b9cdb3da-fdf5-474d-abe8-542ea04d2597)